src/sentry/integrations/vsts/client.py

```diff
@@ -22,20 +22,22 @@ class VstsApiPath(object):
     work_items_create = u'https://{account_name}/{project}/_apis/wit/workitems/${type}'
 
 
-class VstsApiClient(ApiClient):
+class VstsApiClient(OAuth2ApiClient):
     api_version = '4.1'
 
-    def __init__(self, access_token, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
         super(VstsApiClient, self).__init__(*args, **kwargs)
-        self.access_token = access_token
+        if 'access_token' not in self.identity.data:
+            raise ValueError('Vsts Identity missing access token')
 
     def request(self, method, path, data=None, params=None):
+        self.check_auth()
```

Comment:
I really wanted to put this in the `OAuth2ApiClient` class, but I didn't also want to have to go back and alter the headers/url/whatever the subclass using the old identity did.So, I felt this was the best option. Open to suggestions.

Reply:
This looks good to me :)